internal/app/machined/pkg/controllers/cri/cri_test.go

```diff
@@ -0,0 +1,35 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package cri_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"github.com/talos-systems/talos/internal/app/machined/pkg/controllers/cri"
+	"github.com/talos-systems/talos/internal/app/machined/pkg/controllers/ctest"
+)
+
+type CRISuite struct {
+	ctest.DefaultSuite
+
+	seccompProfileDirectory string
+}
+
+func TestSeccompProfileSuite(t *testing.T) {
+	seccompProfiesDirectory := t.TempDir()
+
+	suite.Run(t, &CRISuite{
+		DefaultSuite: ctest.DefaultSuite{
+			AfterSetup: func(suite *ctest.DefaultSuite) {
+				suite.Require().NoError(suite.Runtime().RegisterController(&cri.SeccompProfileController{}))
```

Comment:
this feels like starting both controllers for a single controller test, wdyt @smira

Reply:
fixed